README.md

```diff
@@ -1 +1,23 @@
-# Hello-Microverse-
\ No newline at end of file
+# Hello-Microverse-
+This is a project to help micronauts master Git and Github
+
+# Built With
+Major Languages
+
+# Author
+
+GitHub: @Mugiira-Kim
+Twitter: @Bbm-Cru
+LinkedIn: Mugiirakim
+
+
+# Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+# Show your support
```

Comment:
- [ ] As I previously suggested, I noticed you're not following the Microverse [Template](https://github.com/microverseinc/readme-template/blob/master/README.md) because you used one `#`  instead of two i.e `##` for the
Built With, Author, Contributing, Show your support and Acknowledgments Headers

| Your README  | Microverse README |
| ------------- | ------------- |
| ![Screenshot from 2022-04-20 10-15-38](https://user-images.githubusercontent.com/52934128/164195922-8a988d0f-c82e-4d88-9549-b6a8c1e0b93e.png) | ![Screenshot from 2022-04-20 10-15-20](https://user-images.githubusercontent.com/52934128/164195996-040e4f16-1272-4498-bd14-daf853ff1b56.png)  |

I Kindly suggest you use the Microverse template accordingly